src/vm/functions/mod.rs

```diff
@@ -267,6 +272,27 @@ pub fn lookup_reserved_functions(name: &str, version: &ClarityVersion) -> Option
             SetVar => SpecialFunction("special_set-var", &database::special_set_variable),
             Map => SpecialFunction("special_map", &sequences::special_map),
             Filter => SpecialFunction("special_filter", &sequences::special_filter),
+            BuffToIntLe => NativeFunction(
+                "native_buff_to_int_le",
+                NativeHandle::SingleArg(&conversions::native_buff_to_int_le),
+                // TODO: Create a dedicated cost fn for all BuffToInt-type functions.
```

Comment:
Can you copy this TODO comment to the other instances, too? It'll help us make sure we don't accidentally forget to update one.

Reply:
done